tests/EventListener/JsonRequestBodyValidationSubscriberTest.php

```diff
@@ -122,7 +122,11 @@ public function testValidateRequestBodySkipsValidationWhenRouteIsNotAvailable()
         $request = new Request();
         $request->headers->set('Content-Type', 'application/json');
 
-        $event = new GetResponseEvent($kernelMock, $request, HttpKernelInterface::MASTER_REQUEST);
+        $event = new RequestEvent(
+            $kernelMock,
+            $request,
+            HttpKernelInterface::MASTER_REQUEST
+        );
```

Comment:
As this is actually a recurring piece of code. We can extract it and make it backward compatible:
```php
    /**
     * Creates a request event.
     *
     * @return RequestEvent|GetResponseEvent
     */
    public function createRequestEvent(Request $request)
    {
        $kernelMock = $this->createMock(HttpKernelInterface::class);

        if (class_exists('Symfony\Component\HttpKernel\Event\GetResponseEvent')) {
            return new GetResponseEvent($kernelMock, $request, HttpKernelInterface::MASTER_REQUEST);
        }

        return new RequestEvent($kernelMock, $request, HttpKernelInterface::MASTER_REQUEST);
    }
```

The same approach can be used in `JsonResponseExceptionSubscriberTest`.